opt/scopeserver/dataserver/utils/search.py

```diff
@@ -54,26 +54,39 @@ class ResultTypePair(NamedTuple):
 }
 
 
-def match_result_cost(term: str, result: str) -> Optional[int]:
+def match_result_cost(term: str, result: str, result_type: str) -> Tuple[int, Optional[int]]:
+    term_cost: Optional[int]
```

Comment:
```suggestion
    term_cost: Optional[int] = None
```

And remove the `else` later